src/OrleansProviders/Streams/Memory/IMemoryStreamQueueGrain.cs

```diff
@@ -1,26 +1,22 @@
-﻿using System.Collections.Generic;
+﻿
+using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace Orleans.Providers.Streams.Memory
+namespace Orleans.Providers
 {
     /// <summary>
     /// Interface for In-memory stream queue grain.
     /// </summary>
     public interface IMemoryStreamQueueGrain : IGrainWithGuidKey
     {
         /// <summary>
-        /// Set max event count. The total number of events enqueued can not exceed maxEventCount.
-        /// </summary>
-        Task SetMaxEventCount(int maxEventCount);
```

Comment:
It's not that we will not allow it, as much as there was no usage of it in this case and no clear way for a user to set it.  For instance, it was not configurable, or directly settable.  With no clear story or tests for this case, I concluded it was best not to have the code around until we need it.  Great thing about source control is that the feature is not really gone, we can restore it when it is needed and/or we have the time to fully flesh out the feature.  Unused and untested code is unnecessary maintenance overhead.